requirements.txt

```diff
@@ -6,7 +6,7 @@ requests            # http
 lxml                # html/xml
 openpyxl            # xlsx
 xlrd                # xls
-h5py                # hdf5
+h5py==2.10.0        # hdf5
```

Comment:
We do not generally like to version lock if it can be avoided!

Why is 2.10.0 the preferred version?

Reply:
No worries. I've just removed the pinning on h5py. Thanks!